knitr/curse-dims/curse-dims.Rmd

```diff
@@ -0,0 +1,770 @@
+```
+
+This tells us that 99.9999% of the draws (i.e, 999,999 out of
+1,000,000 draws) from $\mathsf{Binomial}(100, 0.8)$ lie in the range
+$(59, 97)$.  This demonstrates just how atypical the most probable
+sequences are.
+
+#### A Continuous Example of Typicality
+
+Recall that all of the interesting computations in Bayesian statistics
+are expectations (predictions, event probabilities, decision theory
+choice points, etc.).  We can think of the typical set as the smallest
+set that suffices for computing posterior expectations for
+"well-behaved" functions (see the exercises).  That is, if $\Theta$ is
+the support for a density $p(\theta)$, then the typical set $T
+\subseteq \Theta$ has the property that for "well-behaved" functions,
+
+\[
+\mathbb{E}[f(\theta) \, | \, y]
+\ = \
+\int_{\Theta} f(\theta) \, p(\theta \, | \, y) \, \mathrm{d}\theta
+\ \approx \
+\int_{T}  f(\theta) \, p(\theta \, | \, y) \, \mathrm{d}\theta
+\]
+
+That is, it's the set we need to compute expectations.  The fact that
+typical sets exist is why we are able to compute expectations using
+Monte Carlo methods.
+
+We saw in the first plot in section 4 (draws are nowhere near the
+mode) that if we have $D$ i.i.d. draws from a unit normal, that the
+typical set was well bounded away from the posterior mode, which is at
+the origin.  There's no built-in R function to compute the quantiles
+directly, but the next section shows that the distaince from the mode
+reduces to a well-known distribution.
+
+
+## 6. Squared Distance of Normal Draws is Chi-Square
+
+Suppose $y = (y_1, \ldots, y_D) \in \mathbb{R}^D$ and that for each $d
+\in 1{:}D$,
+
+\[
+y_d \sim \mathsf{Normal}(0, 1),
+\]
+
+is drawn independently.  The distribution of the sum of the squared
+elements of $y$ is well known to have a chi-square distribution,
+
+\[
+(y_1^2 + \cdots + y_D^2) \sim \mathsf{ChiSquare}(D).
+\]
+
+Or using ($\mathrm{L}_2$) norm notation, the squared norm of a unit
+normal has a chi-square distribution,
+
+\[
+{|| y ||}^2 \sim \mathsf{ChiSquare}(D).
+\]
+
+This means we could have drawn the curves out by using the inverse
+cumulative distribution function for the chi-square distibution (see
+the exercises).
+
+
+## 7.  Maximum Likelihood and Least Squares
+
+The intimate relation between (squared Euclidean) distance and the
+normal distribution led Gauss to formulate the notion of maximum
+likelihood and show that it reduced to minimizing a sum of squares
+(i.e., "least squares"), when estimating the location parameter of a
+normal distribution.
+
+Suppose we observe $y = (y_1, \ldots, y_N) \in \mathbb{R}^N$ and we
+assume they come from a normal distribution with unit scale $\sigma =
+1$ and unknown location $\mu$.  Then the log density is
+
+\[
+\log p(y \, | \, \mu, 1) 
+\ \propto \
+\sum_{n=1}^N \log \mathsf{Normal}(y_n \, | \, \mu, 1)
+\ \propto \
+-\frac{1}{2} \sum_{n=1}^N \left( y_n - \mu \right)^2
+\ \propto \
+-\frac{1}{2} \sum_{n=1}^N \mathrm{d}(y_n, \mu).
```

Comment:
need to square the `d(y_n, \mu)` term:
-\frac{1}{2} \sum_{n=1}^N \mathrm{d}(y_n, \mu)^2.